cmd/scollector/collectors/snmp_ifaces.go

```diff
@@ -103,9 +103,11 @@ func c_snmp_ifaces(community, host string) (opentsdb.MultiDataPoint, error) {
 		ifTypes[k] = val
 	}
 	for k, v := range ifAliasesRaw {
+		// Replace spaces with underscores for readability, then clean bad characters
+		newAlias := strings.Replace(fmt.Sprintf("%s", v), " ", "_", -1)
```

Comment:
It appears that they used to be, up until they were switched to metadata in 2014 in 665993dcbab0800dc0a5aef8fea47a30a613b651. In the current version of scollector, aliases are only present in metadata. I could add an additional toggle to run this space-to-underscore transform only when set, but anyone upgrading past the older version that had them as tags would see a change in tag behavior/values regardless given that they were later taken away. The immediate impact this change would have on live deployments running an incremental upgrade would be a change in metadata string formatting. I'm happy to support whatever direction you prefer here, just let me know.